staging/src/k8s.io/client-go/features/known_features.go

```diff
@@ -75,9 +81,10 @@ const (
 // After registering with the binary, the features are, by default, controllable using environment variables.
 // For more details, please see envVarFeatureGates implementation.
 var defaultKubernetesFeatureGates = map[Feature]FeatureSpec{
-	ClientsAllowCBOR:        {Default: false, PreRelease: Alpha},
-	ClientsPreferCBOR:       {Default: false, PreRelease: Alpha},
-	InOrderInformers:        {Default: true, PreRelease: Beta},
-	InformerResourceVersion: {Default: false, PreRelease: Alpha},
-	WatchListClient:         {Default: false, PreRelease: Beta},
+	ClientsAllowCBOR:             {Default: false, PreRelease: Alpha},
+	ClientsPreferCBOR:            {Default: false, PreRelease: Alpha},
+	InOrderInformers:             {Default: true, PreRelease: Beta},
+	InOrderInformersBatchProcess: {Default: false, PreRelease: Alpha},
```

Comment:
Can we go to Beta default? I think we want to have feature gate as escape hatch, but feature itself can be enabled by default.